packages/bouncer/extra/dcos_add_user.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
```

Comment:
This docstring discusses the history of this script. But a user of this script just cares about what this script does, and the fact that it continues to do the same thing.

Let's remove all this and change it to `This script adds remote users for the Auth0 identity provider.`

Is anything else really useful here?